src/_vars.scss

```diff
@@ -44,6 +45,9 @@ $wellness-red: #a30000;
 $wellness-green: #005500;
 $wellness-yellow: #ffcc00;
 
+// NOT DEPRECATED
```

Comment:
Good comment, but please say it in a way that will make sense a year from now (after the theme changes are done and forgotten)

Reply:
I think the comments beneath this line are fairly clear, except for a typo I left, I'll commit the fixed typo, hopefully it's clear enough